game/hud/src/components/HUD/index.tsx

```diff
@@ -94,41 +102,47 @@ class HUD extends React.Component<HUDProps, HUDState> {
 
     console.log(e.nativeEvent.deltaY);
 
+    const pos: Position = this.props.layout.widgets[name];
+
     if (e.nativeEvent.deltaY < 0) {
       this.props.dispatch(savePosition(name, {
-        x: this.props.layout.widgets[name].x,
-        y: this.props.layout.widgets[name].y,
-        scale: this.props.layout.widgets[name].scale - factor
+        x: pos.x,
+        y: pos.y,
+        width: pos.width,
+        height: pos.height,
+        scale: pos.scale - factor
       }));
     } else {
       this.props.dispatch(savePosition(name, {
-        x: this.props.layout.widgets[name].x,
-        y: this.props.layout.widgets[name].y,
-        scale: this.props.layout.widgets[name].scale + factor
+        x: pos.x,
+        y: pos.y,
+        width: pos.width,
+        height: pos.height,
+        scale: pos.scale + factor
       }));
     }
   }
 
   draggableWidget = (name: string, widgets: any, Widget: any, containerClass: string, props?: any) => {
-    const w = widgets[name];
+    const pos: Position = widgets[name];
     return (
       <Draggable handle='.drag-handle'
-                  defaultPosition={{x: w.x, y: w.y}}
-                  position={null}
+                  defaultPosition={{x: pos.x, y: pos.y}}
+                  position={{x: pos.x, y: pos.y}}
                   grid={[1, 1]}
                   zIndex={100}
                   onStart={this.onStart}
                   onDrag={this.handleDrag}
                   onStop={(e:any, ui:any) => {
                     this.onStop();
-                    this.props.dispatch(savePosition(name, {x: ui.x, y: ui.y, scale: w.scale}));
+                    this.props.dispatch(savePosition(name, { x: ui.x, y: ui.y, width: pos.width, height: pos.height, scale: pos.scale }));
```

Comment:
this appears to be the only place you have { thing } vs {thing} conform with the rest of the code in this document {thing}?